src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,23 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 6.x` to `io.sentry:sentry-android 7.0.0`
+
+- This version requires API level 19.
+  - Older versions with NDK integration support API level 16.
+    To support the devices on the API level lower than 16, update your `AndroidManifest.xml`:
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <!-- Merging strategy for the imported manifests -->
+    <uses-sdk tools:overrideLibrary="io.sentry.android" />
+</manifest>
+```
+
+    With these changes the SDK will be enabled on all devices with API level ≥ 14 and the SDK with NDK will be enabled on all devices with API level ≥ 16.
+
+- Older versions without NDK integration support API level 14.
```

Comment:
```suggestion
- `io.sentry:sentry-android 7.0.0` requires API level 19.

    Older versions with the NDK integration support API level 16. To support devices on an API level lower than 16, update your `AndroidManifest.xml`:

    ```xml {filename:AndroidManifest.xml}
    <manifest>
        <!-- Merging strategy for the imported manifests -->
        <uses-sdk tools:overrideLibrary="io.sentry.android" />
    </manifest>
    ```

    With these changes the SDK will be enabled on all devices with API level ≥ 14 and the SDK with NDK will be enabled on all devices with API level ≥ 16.

- Older versions without the NDK integration support API level 14.
```

Reply:
formatting tweaks and some small language tweaks